scripts/release.js

```diff
@@ -111,6 +110,29 @@ fs.writeFileSync(
 spawnOrFail('npm', [`version ${versionString} --no-git-tag-version`]);
```

Comment:
Should we also update the peer dependency in the main package.json automatically? We can get the latest version with npm show amazon-chime-sdk-js I think.

Reply:
So we decided to not bump the JS SDK version but test the react demo with the latest right?

Reply:
I thought we said we should update it during each release since we only want to support the latest version of the SDK?

Reply:
Hmm, sorry I am not remembering it correctly, well then in that case we need to update the peer dependency then as Trung mentioned.

Reply:
The release script will update the peer dependency of the main package.json every release.